@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -179,20 +177,24 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
      * Creates the Point of Interest geometries for the given feature.
      *
      * @param layer Tile's layer to be processed.
+     * @param extents Tile's layer extents.
      * @param feature The current feature containing the main geometry.
      * @param env The [[MapEnv]] containing the environment information for the map.
      * @param techniques The array of [[Technique]] that will be applied to the geometry.
      * @param featureId The id of the feature.
      */
     processPointFeature(
         layer: string,
-        geometry: THREE.Vector3[],
+        extents: number,
+        geometry: THREE.Vector2[],
         env: MapEnv,
         techniques: IndexedTechnique[],
         featureId: number | undefined
     ): void {
         this.processFeatureCommon(env);
 
+        const tmpV = new THREE.Vector3();
```

Comment:
you can do that outside the method as well

Reply:
Yeah, this class needs to be rearranged and a lot of allocations can be reused. Will take care of these.